staging/src/k8s.io/apiserver/pkg/apis/apiserver/validation/validation.go

```diff
@@ -333,25 +337,45 @@ func ValidateWebhookConfiguration(fldPath *field.Path, c *api.WebhookConfigurati
 	default:
 		allErrs = append(allErrs, field.NotSupported(fldPath.Child("connectionInfo", "type"), c.ConnectionInfo, []string{"InClusterConfig", "KubeConfigFile"}))
 	}
+	if len(c.MatchConditions) > 64 {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("matchConditions"), c.MatchConditions, "must not exceed 64 matchConditions"))
+	}
+	for i, condition := range c.MatchConditions {
+		fldPath := fldPath.Child("matchConditions").Index(i).Child("expression")
+		if len(strings.TrimSpace(condition.Expression)) == 0 {
```

Comment:
Seems like missing checking for:
- Invalid expression with only spaces
- duplicated matchConditions
- matchCondition name check(using `apivalidation.ValidateQualifiedName`)

Refer to existing validation for matchConditions: https://github.com/kubernetes/kubernetes/blob/d953755686d64207740c15c7eb1599f874f29243/pkg/apis/admissionregistration/validation/validation.go#L917

Reply:
added validation for dups